### PR TITLE
user error and informational console events

### PR DESCRIPTION
**Pull request by shamasis**
This PR simply triggers an additional event when execution error occurs.
It also adds setX and clearX timer functions that log unavailability message in console.

### CODE REVIEW

File: lib/sandbox/index.js [hunk 1]
```diff
@@ -28,5 +28,5 @@ require('./purse');
 
 // Setup the ping-pong and execute routines
 bridge.on('ping', require('./ping').listener('pong'));
-bridge.on('execute', require('./execute').listener('execution.', (typeof console !== 'undefined' ? console : null),
```
Comment (shamasis) [inline lib/sandbox/index.js]: this change is irrelevant to the PR. it is simply a rename to allow easy coding of non legacy handlers down the line

File: lib/sandbox/legacy-execute.js [hunk 2]
```diff
@@ -35,27 +38,32 @@ module.exports = {
...
+            scope.import(new StubTimers(postmanConsole));
             scope.import({
                 postman: postmanLegacyInterface.create(options),
-                console: new PostmanConsole(bridge, cursor, options.debug ? console : null)
+                console: postmanConsole
             });
 
             // do browser specific imports as a special case for jquery
             window && scope.import({
+                XmlHttpRequest: window.XmlHttpRequest,
```
Comment (czardoz) [inline lib/sandbox/legacy-execute.js]: This should be `XMLHttpRequest`

Comment (shamasis) [inline lib/sandbox/legacy-execute.js]: @czardoz fixed

**Comment by shamasis** [comment]
@czardoz i have fixed the xmlhttprequest variable name issue.

**Pull request merged**